tests/acceptance/blocks/verticalresults.js

```diff
@@ -32,19 +35,47 @@ class VerticalResults {
   }
 
   /**
-   * Gets a selector for the card associated with the currently focused map pin.
-   * @returns {Selector}
+   * Returns the number of results from the VerticalResultsCount-start.
+   * @returns {Promise<number>}
    */
-  getFocusedCard () {
-    return this._focusedCard;
+   async getResultsCountStart () {
```

Comment:
Small nit: what do you think of these naming changes
- `getResultsCountTotal` -> `getResultsCount`
- `getResultsCountStart` -> `getResultsOffset`

The phrase `ResultsCountTotal` sounds a bit redundant to me and `ResultsCountStart` is ambiguous.

Reply:
I will update. I think those are clearer